theme/palette/elements.less

```diff
@@ -5,6 +5,7 @@
 @sk-highlight: #1d9ede;
 @sk-footer: @sk-mid-gray-light;
 @sk-background: @sk-gray-light;
+@sk-card-bg: @sk-white;
```

Comment:
should we then replace it in a card component as well? Feels very disconnected though, because card might be transparent as well 🤔

Reply:
Most of the colour organisation work is waiting for proper CSS-in-JS. `@sk-card-bg` was an interim measure suggestion from @michaeltaranto. We also want to add `@sk-body` but that can wait for the colour re-org